Obsidian_Runtime/src/main/yul_templates/object.mustache

```diff
@@ -16,26 +16,34 @@ object "{{creationObject}}" {
             {{#constructorHasParams}} let {{constructorParams}} := {{copyConstructorArguments}}() {{/constructorHasParams}}
             {{implicitConstructor}}({{constructorParams}})
         {{/notLibrary}}
-        {{! call constructor }}
+        {{! todo: write and call constructor }}
         {{#deploy}}
             {{call}}
         {{/deploy}}
         {{! functions related to constructor }}
         {{#deployFunctions}}
             {{code}}
         {{/deployFunctions}}
+        {{codeCopy}}
```

Comment:
What is this for?

Reply:
it's a cleaner way to insert the code copy line than trying to insert it in to a traversal of the obsidian ast.